app/DynamicsAdapter/Fams3Adapter.Dynamics/Types/TelephoneNumberType.cs

```diff
@@ -12,6 +12,7 @@ public class TelephoneNumberType : Enumeration
         public static TelephoneNumberType Home = new TelephoneNumberType(867670001, "Home");
         protected TelephoneNumberType(int value, string name) : base(value, name)
```

Comment:
only 1 type?

Reply:
There are more than one See actual file for detail. That must have been an update that was seen as a change